DOCUMENTATION.md

```diff
@@ -317,7 +324,7 @@ fs.readFile('path/to/document.pdf', function (err, pdfFile) {
       application_date: '2018-01-01',
       link_to_file: notifyClient.prepareUpload(pdfFile)
     }
-  }).then(response => console.log(response.body)).catch(err => console.error(err))
```

Comment:
I've removed the `body` from `response` here and also on line 351 because none of our other response handling actually looks at any properties of `response`, they all just `console.log(response)` and so I've made this change for consistency.

Reply:
These are the only places where we mention the use of `response.body` to access the response data so there's a good chance users of our docs figured this out from those examples.

If we remove `.body` (or `.data` now) from here, we should ensure it is mentioned in any of the sections headed 'Response' (ie. https://github.com/alphagov/notifications-node-client/blob/master/DOCUMENTATION.md#response-1)

Reply:
Good point. I've taken a stab at a sentence that hopefully makes it clear that they need to look at `response.data` rather than looking just at `response` and added this in a new commit